market/connector/docusign/demo/eSignatureDemoTest/src_test/net/docusign/test/TestDocuSignDemo.java

```diff
@@ -4,20 +4,38 @@
 
 import org.junit.jupiter.api.Test;
 
+import ch.ivyteam.ivy.application.IApplication;
 import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
 import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
 import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
 import ch.ivyteam.ivy.environment.AppFixture;
 import ch.ivyteam.ivy.security.ISession;
+import ch.ivyteam.ivy.workflow.ITask;
+import ch.ivyteam.ivy.workflow.WorkflowNavigationUtil;
 
 @IvyProcessTest
 public class TestDocuSignDemo
 {
   @Test
-  public void main(BpmClient bpmClient, ISession session, AppFixture fixture)
+  public void main(BpmClient bpmClient, ISession session, AppFixture fixture, IApplication app)
   {
     fixture.environment("dev-axonivy");
     
+    ExecutionResult result = userFlow(bpmClient, session);
+    net.docusign.esignature.demo.Data docuSign = result.data().last();
+    assertThat(docuSign.getEnvelopes()).hasSize(1);
+    
+    ITask wait4Signing = result.workflow().activeTasks().get(0);
+    fireIntermediateEvent(app, wait4Signing, docuSign.getEnvelopeId());
+    
+    ExecutionResult endResult = bpmClient.start().task(wait4Signing).as().systemUser().execute();
+    assertThat(endResult.bpmError()).isNull();
+//    ICase activeCase = endResult.workflow().activeCase();
+//    assertThat(activeCase.documents().getAll()).isNotEmpty();
```

Comment:
are this commits wanted?

Reply:
yes, it documents a shortcoming of the current bpmClient test which I maybe do not find time to address myself before vacation. So I kept it a little more obvious that this test doesn't really assert what it should.
The whole use-case is about having signed-documents attached to a case ... but right now I can't drive the test that far > https://jira.axonivy.com/jira/browse/XIVY-5373